paasta_tools/check_chronos_jobs.py

```diff
@@ -264,6 +277,19 @@ def sensu_message_status_for_jobs(chronos_job_config, service, instance, cluster
     return output, sensu_status
 
 
+def job_expected_runtime_callback(client, chronos_job):
+    """Returns a function that is supposed to be called from job_is_stuck() to get the job expected runtime."""
+    def f():
+        job_expected_runtime = 0
+        if chronos_job:
+            try:
+                job_expected_runtime = int(client.job_stat(chronos_job['name'])['histogram']['99thPercentile'])
+            except KeyError:
+                paasta_print(utils.PaastaColors.yellow("Can't get 99thPercentile for %s." % chronos_job['name']))
```

Comment:
yea, good old normal logging for this one. paasta_prints are usually for user-facing output